_layouts/blog.html

```diff
@@ -12,6 +12,19 @@
       <div class="row pt-4 d-flex align-items-center text-center h-100">
         <div class="col-12 text-white">
           <h1 class="display-3 w-100"> {{ page.title }} </h1>
+          <h4>
+            {% for author in page.authors %}
+              {% if forloop.first %} by {% endif %}
+              {% if author.email %}<a href="mailto:{{ author.email }}">{% endif %}
+              {{ author.name }}
```

Comment:
What if `author.name` is not supplied?

Reply:
If there is no `authors` this should work. If not providing a name is actually a concern I can simply add a default. Suggestions?

Reply:
No I meant actually some kind of error case where the author e.g. only provides the other keys or misspells the name key.

Reply:
Jekyll would not fail, it will be empty. Any preference on a placeholder?

Reply:
"Anonymous"?

Reply:
Added.